radicle/src/storage/git.rs

```diff
@@ -143,20 +143,26 @@ impl Storage {
         self.path.as_path()
     }
 
-    pub fn projects(&self) -> Result<Vec<Id>, Error> {
-        let mut projects = Vec::new();
+    pub fn repositories(&self) -> Result<Vec<Id>, Error> {
+        let mut repos = Vec::new();
```

Comment:
Just curious, why the change from project -> repository?

Reply:
Project is incorrect/overly specific. "Project" is a term for repositories that have a project payload. But here we're just returning all repositories under storage.